rpc/wrtc_client_channel.go

```diff
@@ -155,6 +155,7 @@ func (ch *webrtcClientChannel) invoke(
 				if clientStream.trailers != nil {
 					*optV.TrailerAddr = clientStream.trailers.Copy()
 				}
+			case grpc.StaticMethodCallOption:
```

Comment:
I think we'll want a comment above this `case` stating why we're accepting and ignoring this call option.

Reply:
LGTM as long as we get a comment in here.